infer/tests/codetoanalyze/c/bufferoverrun/main.c

```diff
@@ -0,0 +1,23 @@
+void trivial();
+void function_call();
+void for_loop();
+void while_loop();
+void do_while();
+void nested_loop();
+void nested_loop_with_label();
+int break_continue_return();
+void goto_loop();
+void inf_loop();
+
+int main() {
```

Comment:
why do you need this, doesn't analyzer work without `main`? What happens if you rename `main` to `run`?